frontend/src/pages/device/index.vue

```diff
@@ -44,7 +75,9 @@ import { Roles } from '@core/lib/roles'
 
 // components
 import NavItem from '@/components/NavItem'
-import SectionTopMenu from '@/components/SectionTopMenu'
+import InstanceStatusHeader from '@/components/InstanceStatusHeader'
```

Comment:
Perhaps better for a wider discussion, but I'd strongly prefer we start using relative paths. 

The magic `@` locks us to webpack and prevents lots of tooling from working correctly, including VSCodes "open file" shortcuts.

Reply:
Although isn't the "@" alias [part of vitest](https://github.com/flowforge/flowforge/blob/24d82a7f4b7640645eaaef397ab9c32ccd2dabed/config/vitest.config.ts#L22) rather than webpack?

Reply:
It looks like vitest had to be told about the webpack alias to be able to resolve files properly. 

The aliases are set up in [config/webpack.config.js](https://github.com/flowforge/flowforge/blob/152ee7e1f01b01f28e395db1219cde80df862881/config/webpack.config.js#L127-L129)